crates/schema/src/auto_migrate.rs

```diff
@@ -74,6 +79,94 @@ impl<'def> MigratePlan<'def> {
     }
 }
 
+/// A migration policy that determines whether a module update is allowed to break client compatibility.
+///
+/// `Compatible` requires migration to maintain backward compatibility.
+/// `BreakingChange` allows breaking changes but requires a valid `MigrationToken`.
+#[derive(Debug, Clone, PartialEq, Eq)]
+pub enum MigrationPolicy {
+    Compatible,
+    BreakClients(spacetimedb_lib::Hash),
```

Comment:
Please add doc comment describing this value and where to obtain it, like in the client-api crate.